server/protocols/webadmin/webadmin-data/src/main/java/org/apache/james/webadmin/routes/AliasRoutes.java

```diff
@@ -86,7 +100,29 @@ public String getBasePath() {
 
     @Override
     public void define(Service service) {
-        service.put(USER_IN_ALIAS_SOURCES_ADDRESSES_PATH, this::addToAliasSources);
+        service.get(ROOT_PATH, this::listAddressesWithAliases, jsonTransformer);
+        service.get(ALIAS_ADDRESS_PATH, this::listAliasesOfAddress, jsonTransformer);
+        service.put(USER_IN_ALIAS_SOURCES_ADDRESSES_PATH, this::addAlias);
+        service.delete(USER_IN_ALIAS_SOURCES_ADDRESSES_PATH, this::deleteAlias);
+    }
+
+    @GET
+    @Path(ROOT_PATH)
+    @ApiOperation(value = "getting addresses containing aliases list")
+    @ApiResponses(value = {
+        @ApiResponse(code = HttpStatus.OK_200, message = "OK", response = List.class),
+        @ApiResponse(code = HttpStatus.INTERNAL_SERVER_ERROR_500,
+            message = "Internal server error - Something went bad on the server side.")
+    })
+    public ImmutableSet<String> listAddressesWithAliases(Request request, Response response) throws RecipientRewriteTableException {
+        return recipientRewriteTable.getAllMappings()
```

Comment:
IMO we should have a `RRT.getDestinationsForType(type.ALIAS)` ?

This would allow RRT implementations to 

 1. Optimize that 'now exensive' operation
 2. Test it specifically

This would be the symetric operation of `getSourcesForType` thus it makes sens to include it in the RRT API even if we get a single call.